backend/src/routes.js

```diff
@@ -9,21 +10,25 @@ import {
   updateCard,
   shareCard,
   unshareCard,
-} from './api/cards.js'
+  getAllCardsWithTasks,
+} from './api/cards.js';
 
-const router = express.Router()
+const router = Router();
 
 // tasks
-router.get('/tasks', getUserTasks)
+router.get('/tasks', getUserTasks);
+router.post('/tasks', createTask);
+router.get('/tasks-with-cards', getTasksWithCards);  // Новый маршрут для получения задач с карточками

```

Comment:
There are no needed. But if we need to create such endpoint - we could expand the existed one 😉(Not add a new one for each feature or interaction with entity)